core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -273,25 +273,50 @@ public void trackMetric(MetricTelemetry telemetry) {
      * @param properties Named string values you can use to search and classify trace messages.
      * @param metrics Measurements associated with this exception event. Appear in "custom metrics" in Metrics Explorer.
      */
-    public void trackException(Exception exception, Map<String, String> properties, Map<String, Double> metrics) {
+    public void trackException(Throwable exception, Map<String, String> properties,
```

Comment:
It's a good move to make this signature to accept Throwable. I think this is also not a breaking change.